src/audio/types.h

```diff
@@ -61,6 +61,7 @@ class ChannelCount {
             return 2;
         }
         DEBUG_ASSERT(!"unreachable code");
+        return 2;
     }
 
```

Comment:
fyi, this is technically not unreachable because its defined that an enum may have any value that can be fit in a type that is at least as wide as the type required to store all the members of the enum. Since the enum has 3 members, the underlying type is two bits large, so thats why (usually) MSVC is warning here.  

That said, we already have `kValueDefault`, so it probably makes more sense to use that instead of assuming stereo...

Reply:
> Since the enum has 3 members, the underlying type is two bits large. 

Not typed enum classes are int (32 bit).

Reply:
Yes, the underlying type usually is. I was talking about this pre-C++14 behavior where it was valid for values of an enum class to be outside the actual enum range. https://www.open-std.org/jtc1/sc22/wg21/docs/cwg_defects.html#1766